index.js

```diff
@@ -16,46 +20,71 @@ var defaultOptions = {
 module.exports = {
   name: 'ember-bootstrap',
 
-  included: function included(app) {
+  included(app) {
```

Comment:
Btw, already did some changes on the master branch that also moved to ES6 syntax, see #208. These are some things I wanted to do also in preparation for the BS4 changes...